src/components/calculator.js

```diff
@@ -1,57 +1,48 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import './calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+const Calculator = () => {
```

Comment:
The Calculator component was refactored from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html) 🔥